### PR TITLE
Adds delay between game and leaderboard

### PR DESCRIPTION
**Pull request by RustyMonky**

### CODE REVIEW

File: screens/main/scoreDisplay.gd [hunk 1]
```diff
@@ -9,36 +9,36 @@ var timer_value = GameData.get("timer_max")
...
-			main.show_leaderboard()
\ No newline at end of file
+    if (GameState.get_state() != GameState.LEADERBOARD && GameState.get_state() != GameState.INITIALS):
+        if (timer_value != 0):
+            timer_value -= 1
+            GameData.store("current_time", timer_value)
+            update_timer_display(timer_value)
+        else:
+            timer.stop()
+            get_parent().get_parent().get_node("leaderboardTimer").start()
```
Comment (sprngr) [inline screens/main/scoreDisplay.gd]: The main var used in the removed code is a reference to `get_parent().get_parent()` so it was easier to manage that reference.

**Comment by sprngr** [comment]
What happened with all the whitespace in this? I'm concerned it may cause conflicts with anything else checked out at this time.

**Comment by RustyMonky** [comment]
@sprngr valid concern -- honestly this happens whenever I pull down master after what I believe is one of travis' commits. I personally did not update any of the file spacing on my own...

**Comment by travm** [comment]
Looks like you boys need to fix your editors. ;)

**Comment by sprngr** [comment]
Just make sure everyone has their Godot editor set to this:

File: screens/main/scoreDisplay.gd (hunk 1 above)
Comment (travm) [reply screens/main/scoreDisplay.gd]: @sprngr Is this good to go?

**Comment by travm** [comment]
I'm still accidentally blasting through the initials. Can we increase the delay even more? And also, can we allow the player to go left or right to edit the characters they are inputting in case they accidentally enter in the first character? Just in case the player still accidentally inputs something, they will be able to edit it.

**Pull request merged**